x-pack/plugins/apm/ftr_e2e/cypress/e2e/read_only_user/deep_links.cy.ts

```diff
@@ -11,7 +11,7 @@ describe('APM deep links', () => {
   });
   it('navigates to apm links on search elastic', () => {
     cy.visitKibana('/');
-    cy.getByTestSubj('nav-search-input').type('APM');
+    cy.getByTestSubj('nav-search-input').type('APM').focus();
```

Comment:
It's odd we have to manually focus the input field (should happen automatically when using `type`)